### PR TITLE
Add Makefile target to restart the forum service

### PR DESCRIPTION
**Pull request by alanoe**

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @alanoe! I've created [OSPR-4363](https://openedx.atlassian.net/browse/OSPR-4363) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).

**Comment by natabene** [comment]
@alanoe Thank you for your contribution. Are you submitting as an individual or on behalf of your employer?

**Comment by alanoe** [comment]
@natabene You're welcome. As an individual.

**Comment by natabene** [comment]
@alanoe Thanks for letting me know. Same as in the other PR, we can review once we receive your signed contributor agreement, provided in the other PR.

**Comment by alanoe** [comment]
@natabene I had already signed it and sent it by email and was just waiting the answer from Legal. I got it today, so this blocker is out of the way.

**Comment by natabene** [comment]
@alanoe Thank you for joining the Open edX community! We have added your user to our records and can now review your code going forward.

File: Makefile [hunk 1]
```diff
@@ -277,6 +277,9 @@ lms-update-db: ## Run migrations LMS container
 
 update-db: | studio-update-db lms-update-db discovery-update-db ecommerce-update-db credentials-update-db ## Run the migrations for all services
 
+forum-restart:
```
Comment (kdmccormick) [inline Makefile]: Can you add a `##` comment for the `make help` message?

**Review by kdmccormick — COMMENTED** [review]
Nice. This is relevant to [my reply to your comment about `%-restart`](https://github.com/edx/devstack/pull/505#discussion_r407666383). Once we reach a consensus there, I'll give this a thumbs up.

File: Makefile (hunk 1 above)
Comment (alanoe) [reply Makefile]: @kdmccormick Done!

**Review by kdmccormick — APPROVED** [review]
Cool. Merging without tests passing because this couldn't possibly affect them.

**Comment by openedx-webhooks** [comment]
@alanoe 🎉 Your pull request was merged!

Please take a moment to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=edx/devstack&entry.867055334=https://github.com/edx/devstack/pull/512&entry.1484655318=https://github.com/alanoe&entry.752974735=2020-04-07+18:51&entry.1917517419=2020-04-20+16:00&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**